Admin/RouteAdmin.php

```diff
@@ -117,11 +109,6 @@ public function setContentRoot($contentRoot)
         $this->contentRoot = $contentRoot;
     }
 
-    public function setControllerResolver($controllerResolver)
```

Comment:
hm, this is a BC break for the SimpleCmsBundle: https://travis-ci.org/symfony-cmf/SimpleCmsBundle/jobs/48334145

what should we do? re-add an empty method?